dashboard/app/models/regional_partners_school_district.rb

```diff
@@ -0,0 +1,39 @@
+# == Schema Information
+#
+# Table name: regional_partners_school_districts
+#
+#  regional_partner_id :integer          not null
+#  school_district_id  :integer          not null
+#
+# Indexes
+#
+#  index_regional_partners_school_districts_on_regional_partner_id  (regional_partner_id)
+#  index_regional_partners_school_districts_on_school_district_id   (school_district_id)
+#
+
+class RegionalPartnersSchoolDistrict < ActiveRecord::Base
+  belongs_to :regional_partner
+  belongs_to :school_district
+
+  CSV_HEADERS = {
+    school_district_id: 'LEAID',
+    regional_partner_name: 'RegionalPartner',
+  }.freeze
+
+  CSV_IMPORT_OPTIONS = { col_sep: "\t", headers: true}
+
+  NO_PARTNER = 'NO PARTNER'
+
+  def self.find_or_create_all_from_tsv(filename)
+    CSV.read(filename, CSV_IMPORT_OPTIONS).each do |row|
+      regional_partner_name = row[CSV_HEADERS[:regional_partner_name]]
+      next if regional_partner_name == NO_PARTNER
+      school_district_id = row[CSV_HEADERS[:school_district_id]]
+
+      regional_partner = RegionalPartner.where(name: regional_partner_name).first
```

Comment:
What if it's null? Should we log or raise an error? Or just assign null below (as it's doing now)?